SPECS/conda/conda.spec

```diff
@@ -0,0 +1,436 @@
+# The tests change a file to read-only and expect an exception when trying to open the file for appending
+# The exception does occur for normal users but not for root 
+# Since root is required for pytest and build tools, skip:
+# tests/core/test_package_cache_data.py::test_instantiating_package_cache_when_both_tar_bz2_and_conda_exist_read_only \
+# tests/gateways/disk/test_delete.py::test_remove_file \
+# tests/gateways/disk/test_delete.py::test_remove_file_to_trash \
+# tests/gateways/disk/test_permissions.py::test_make_writable \
+# tests/gateways/disk/test_permissions.py::test_recursive_make_writable \
+# tests/gateways/disk/test_permissions.py::test_make_executable 
+
+
+mkdir /tmp
+
+pip3 install atomicwrites>=1.3.0 \
+    attrs>=19.1.0 \
+    ruamel.yaml>=0.11.14 \
+    more-itertools>=7.0.0 \
+    pluggy>=0.11.0 \
+    pytest>=5.4.0 \
+    pytest-cov>=2.7.1 
+PATH=%{buildroot}%{_bindir}:${PATH} \
+PYTHONPATH=%{buildroot}%{python3_sitelib} \
+    python%{python3_version} -m pytest -vv -m "not integration" \
+    --deselect=tests/test_cli.py::TestJson::test_list \
+    --deselect=tests/test_cli.py::TestRun::test_run_returns_int \
+    --deselect=tests/test_cli.py::TestRun::test_run_returns_nonzero_errorlevel \
+    --deselect=tests/test_cli.py::TestRun::test_run_returns_zero_errorlevel \
+    --deselect=tests/core/test_package_cache_data.py::test_ProgressiveFetchExtract_prefers_conda_v2_format \
+    --deselect=tests/core/test_subdir_data.py::test_subdir_data_prefers_conda_to_tar_bz2 \
+    --deselect=tests/core/test_subdir_data.py::test_use_only_tar_bz2 \
+    --deselect=tests/core/test_initialize.py \
+    --deselect=tests/core/test_solve.py::test_cuda_fail_1 \
+    --deselect=tests/base/test_context.py::ContextCustomRcTests::test_target_prefix \
+    --deselect=tests/core/test_package_cache_data.py::test_instantiating_package_cache_when_both_tar_bz2_and_conda_exist_read_only \
+    --deselect=tests/gateways/disk/test_delete.py::test_remove_file \
+    --deselect=tests/gateways/disk/test_delete.py::test_remove_file_to_trash \
+    --deselect=tests/gateways/disk/test_permissions.py::test_make_writable \
+    --deselect=tests/gateways/disk/test_permissions.py::test_recursive_make_writable \
+    --deselect=tests/gateways/disk/test_permissions.py::test_make_executable 
+
+
+%files
+%{_bindir}/conda
+%{_bindir}/conda-env
+%{bash_completionsdir}/conda
+# TODO - better ownership/requires for fish
+%dir /etc/fish
+%dir /etc/fish/conf.d
+/etc/fish/conf.d/conda.fish
+/etc/profile.d/conda.sh
+/etc/profile.d/conda.csh
+
+%files -n python%{python3_pkgversion}-conda
+%license LICENSE.txt
+%doc CHANGELOG.md README.rst
+%{python3_sitelib}/conda/
+%{python3_sitelib}/conda_env/
+%{python3_sitelib}/*.egg-info
+%exclude %{python3_sitelib}/test_data/
+%{_localstatedir}/cache/conda/
+%{_datadir}/conda/
+
+
+%changelog
+* Wed Jun 23 2021 Rachel Menge <rachelmenge@microsoft.com> - 4.10.1-2
+- Initial CBL-Mariner version imported from Fedora 34 (license: MIT)
```

Comment:
Please always check the license for each package and indicate that with `License verified` or a similar phrase (makes it easier to check where we might have missed checking the license).